src/main/scala/scalalab3/chatbotengine/telegram/PoolingEngineActor.scala

```diff
@@ -0,0 +1,60 @@
+package scalalab3.chatbotengine.telegram
+
+import java.util.concurrent.TimeUnit
+
+import akka.actor.{Actor, ActorLogging, ActorRef, Props}
+import akka.pattern.pipe
+import com.typesafe.config.ConfigFactory
+
+import scala.concurrent.duration.Duration
+
+import scalalab3.chatbotengine.core.ChatBot
+import scalalab3.chatbotengine.model.{OutMessage, Update}
+
+object PoolingEngineActor {
+  def props(bots: Traversable[ChatBot]): Props = Props(new PoolingEngineActor(bots))
+
+  case class Tick(ref: ActorRef)
+  case class BatchUpdate(dest: ActorRef, payload: Seq[Update])
+}
+
+class PoolingEngineActor(bots: Traversable[ChatBot]) extends Actor with ActorLogging {
+  import PoolingEngineActor._
+  import context.dispatcher
+
+  val registry = bots.map { b =>
+    val cfg = ConfigFactory.load()
+    val auth = cfg.getString(s"bots.${b.getClass.getSimpleName}")
+    val token = cfg.getString(s"tokens.$auth")
+    val client = new TelegramClient(auth, token)
+
+    val bot = context.actorOf(BotActor.props(b))
+    (bot, client)
+  }.toMap
+
+  registry.keys.foreach { b =>
+    context.system.scheduler.schedule(
+      Duration.create(1, TimeUnit.SECONDS), Duration.create(1, TimeUnit.SECONDS),
```

Comment:
More elegant way is use "1.second" (import import scala.concurrent.duration._ as I remember)